uber/configspec.ini

```diff
@@ -44,6 +44,11 @@ event_qr_id = string(default='')
 # Events who don't have barcode scanners will want to turn this off.
 use_checkin_barcode = boolean(default=True)
 
+# This controls whether the promo code field will be visible on the
+# registration form. True to allow attendees to use promo codes when
+# registering, False to disallow promo codes.
+promo_codes_enabled = boolean(default=True)
```

Comment:
I'd prefer to call this badge_promo_codes_enabled or something, to avoid future confusion vs. merch promo codes if we add those in the future.